01week/datatypes.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+// Write a JavaScript program to display the current day and time.
+
+const currentDate = new Date();
+const currentDateAndTime = (currentDate.getMonth() + 1) + '/' + currentDate.getDate() + '/' + 
+currentDate.getFullYear();
+
+console.log(currentDateAndTime);
+
+// Write a JavaScript program to convert a number to a string.
+
+const convertToString = (num1) => num1.toString();
```

Comment:
this needs to be a function that takes in any number and returns the number converted as a string